src/upload_s3.py

```diff
@@ -0,0 +1,55 @@
+import logging
+import os
+import argparse
+import glob
+import boto3
+
+logging.basicConfig(format="%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
+
+def get_file_names(top_dir, file_expr):
```

Comment:
would detailed doc strings with args and returns be necessary for "smaller functions"?

Reply:
never hurts